lib/dataTrigger/dpnsTriggers/domainCreateDataTrigger.js

```diff
@@ -0,0 +1,63 @@
+const DataTriggerExecutionResult = require('../DataTriggerExecutionResult');
+const DataTriggerExecutionError = require('../../errors/DataTriggerExecutionError');
+const hash = require('../../util/hash');
+const Document = require('../../document/Document');
+const DataTrigger = require('../DataTrigger');
+
+/**
+ * @typedef domainCreateDataTrigger
+ * @param {Document} document
+ * @param {DataTriggerExecutionContext} context
+ * @return {Promise<DataTriggerExecutionResult>}
+ */
+async function domainCreateDataTrigger(document, context) {
+  const dataTriggerExecutionResult = new DataTriggerExecutionResult();
+
+  if (document.getAction() !== Document.ACTIONS.CREATE) {
```

Comment:
I think we don't need to check it inside triggers. It should be verified outside.

Reply:
Already fixed